roles/network_plugin/calico/rr/tasks/update-node.yml

```diff
@@ -14,11 +14,12 @@
     delay: "{{ retry_stagger | random + 3 }}"
     retries: 10
 
-  - name: Calico-rr | Set route reflector cluster ID
+  - name: Calico-rr | Set route reflector cluster ID and label
     set_fact:
       calico_rr_node_patched: >-
         {{ calico_rr_node.stdout | from_json | combine({ 'spec': { 'bgp':
-        { 'routeReflectorClusterID': cluster_id }}}, recursive=True) }}
+        { 'routeReflectorClusterID': cluster_id }}}, recursive=True) |
+        combine({ 'metadata': { 'labels': { 'i-am-a-route-reflector': 'true' }}}, recursive=True) }}
```

Comment:
The original task `Calico-rr | Set label for route reflector` retries 10 times, but this task doesn't retry.
Even if adding setting of the label, this is task stabler on your tests, right?

Reply:
Hi @oomichi @floryut  , sorry for the delay... the original task needed retries because this one reloads calico. The thing is, if we perform the changes in the same operation, there is no need for retries. There is no second task that needs to wait for the first one.